extension/src/experiments/model/collect.ts

```diff
@@ -86,7 +86,7 @@ const getDisplayNameOrParent = (
     return `(${shortenForLabel(checkpointParent)})`
   }
   if (name) {
-    return `[${name}]`
```

Comment:
Is there any possible issues with taking off the brackets of the experiment `displayNameOrParent` that I missed?